javascript/javascript2/week1/countdanishLetters.js

```diff
@@ -0,0 +1,29 @@
+//const danishString = "Jeg har en blå bil";
+const danishLetters="å,ø,æ";
+function checkDanishLetters(string){
+    let charå=0;
+    let charø=0;
+    let charæ=0;
+    let returnObject={};
+    for(let i=0;i<string.length;i++){
+        let char=string[i];
+if(char==='å'){
+    
+    charå++;
+}
+if(char==='ø'){
+    charø++;
+    }
+    if(char==='æ'){
+        charæ++;
+    }
+    returnObject={total:charå+charø+charæ ,å:charå,ø:charø,æ:charæ };
+        
+} 
+return returnObject;
+}
+const danishString = "Jeg har en blå bil";
+console.log(checkDanishLetters(danishString)); // returns {total: 1, å: 1}
```

Comment:
this line outputs:
`{ total: 1, 'å': 1, 'ø': 0, 'æ': 0 }` not `{total: 1, å: 1}`
How can you change your code to not show `0` entries?